org.testeditor.web.dropwizard/src/test/java/org/testeditor/web/dropwizard/AbstractIntegrationTest.xtend

```diff
@@ -0,0 +1,35 @@
+package org.testeditor.web.dropwizard
+
+import de.xtendutils.junit.AssertionHelper
+import io.dropwizard.testing.ResourceHelpers
+import io.dropwizard.testing.junit.DropwizardAppRule
+import javax.ws.rs.client.Invocation.Builder
+import org.junit.Rule
+
+import static io.dropwizard.testing.ConfigOverride.config
+
+abstract class AbstractIntegrationTest {
+
+	val configs = #[
+		config('server.applicationConnectors[0].port', '0')
+	]
+
+	@Rule
+	public val dropwizardAppRule = new DropwizardAppRule(
+		ExampleApplication,
+		ResourceHelpers.resourceFilePath('test-config.yml'),
+		configs
+	)
+
+	protected extension val AssertionHelper = AssertionHelper.instance
+	protected val client = dropwizardAppRule.client
+	var token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImpvaG5kb2UiLCJuYW1lIjoiSm9obiBEb2UiLCJlbWFpbCI6ImpvaG5AZXhhbXBsZS5vcmcifQ.IIyw2DMeoXfZ6AIiRirkm-f-rLmsKpVGaNp7-KN1Jyw'
```

Comment:
comment about actual json that was encoded into jwt? this will give the derived tests an idea of what user to expect. (or even better encode json into token here, if applicable)